tests/integration/ha/continuous_writes.py

```diff
@@ -26,6 +27,14 @@
 logger = logging.getLogger(__name__)
 
 
+@dataclass
+class ContinuousWritesResult:
+    count: int = -1
+    last_message: Optional[object] = None
```

Comment:
**niptick** maybe `Optional[dict | int]`? `object` is a rather strange typing...

Reply:
I know, the result can be some custom classes from the Kafka python API, wich are obscured by the main class being a Generator, it makes digging up the type a bit harder.

Reply:
```python
Generator[Union[Dict[str, str], int]]
```

doesn't work here?

Reply:
Or
```python
Iterator[]
```

Reply:
Yeah, but the message itself is neither an iterator nor generator, so the typing would be quite misleading

Reply:
I think that's fine though, as you type-hint a generator as an iterable that yields a type, right?

Reply:
I just changed to be `ConsumerRecord`. Its a bit weird since it's used kind of internally on the kafka-python. But at least it's the correct type